src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -900,6 +906,15 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
                     coherencyUpdate.deps.Select(du => du.To).ToList(), message.ToString());
             }
 
+            // If the coherency algorithm failed and there are no non-coherency updates and
+            // we crate an empty commit that describes an issue.
+            if (requiredUpdates.Count == 0)
+            {
+                string message = "The Strict Coherency algorithm failed";
```

Comment:
```suggestion
                string message = "Failed coherent parent dependency update.";
```

I don't think we need to mention strict here, since strict becomes the only option.